### PR TITLE
Add support for complex to the power of real

### PR DESCRIPTION
**Pull request by lucasgruwez**
Previously, a complex number to the power of a real number (e.g. [2 + 3i]^2 ) would throw an error:

```
[TheoremJS]: Complex operation require complex numbers
```

I implemented a quick cheat for this. If the argument for `complex.pow()` is complex, then the code runs as usual. If the argument is a number, it runs the code with the complex number `n + 0i`. Else, it throws a type error: 

```
[TheoremJS]: Unsupported typeof power
```

### CODE REVIEW

File: src/includes/functions/math/complex/src/pow.js [hunk 1]
```diff
@@ -17,9 +17,6 @@ pow(complex) {
 	 * Im = exp(c * logsq2 - d * arg(z_1)) * sin(d * logsq2 + c * arg(z_1))
 	 *
 	 */
-	if (!complex.isComplex) {
-		throw "[TheoremJS]: Complex operation require complex numbers"
-	}
```
Comment (lucasgruwez) [inline src/includes/functions/math/complex/src/pow.js]: Remove previous `throw err`, as we are now going to support more than just complex arguments

File: src/includes/functions/math/complex/src/pow.js [hunk 2]
```diff
@@ -66,11 +63,21 @@ pow(complex) {
...
+
+	if (complex.isComplex) {
+    	c = complex.a;
+    	d = complex.b;
+	} else if (typeof complex === 'number') {
+		c = complex;
+		d = 0;
+	} else {
+		throw "[TheoremJS]: Unsupported typeof power"
+	}
```
Comment (lucasgruwez) [inline src/includes/functions/math/complex/src/pow.js]: If argument is a complex number, run as before, if `typeof` the argument is a number, let `real = n` and `im = 0`, else throw type error.

**Comment by lucasgruwez** [comment]
Looks like your test is failing, although it sais 76 passes 0 fails 🤔.
Maybe a problem with eye.js ¯\_(ツ)_/¯

**Comment by arguiot** [comment]
Alright, I'll fix the build (the issue is with GitHub actions, but I found a way to fix it) and I'll try to add some tests

**Comment by arguiot** [comment]
@lucasgruwez I've submitted you a [PR](https://github.com/lucasgruwez/TheoremJS/pull/1), could you take a look please?

**Comment by lucasgruwez** [comment]
I've merged your PR!

**Pull request merged**